apt-hook/json-hook.cc

```diff
@@ -236,18 +262,20 @@ bool collect_pro_packages_from_pre_prompt_json(json_object *pre_prompt, std::vec
 #define MAX_COUNT_MESSAGE_LEN 256
 std::string create_count_message(security_package_counts &counts) {
     char buf[MAX_COUNT_MESSAGE_LEN] = {0};
+    release_info info = get_release_info();
+    std::string lts_text = info.is_lts ? "LTS " : "";
 
     if (counts.esm_apps == 0) {
         if (counts.esm_infra == 0) {
             if (counts.standard == 0) {
                 return "";
             } else if (counts.standard == 1) {
-                return std::string(gettext("1 standard LTS security update"));
+                return std::string(gettext("1 standard ")) + lts_text + gettext("security update");
             } else if (counts.standard > 1) {
                 std::snprintf(
                     buf,
                     MAX_COUNT_MESSAGE_LEN,
-                    gettext("%lu standard LTS security updates"),
+                    (std::string(gettext("%lu standard ")) + lts_text + gettext("security updates")).c_str(),
                     counts.standard
```

Comment:
We should insert the "LTS " via string formatting, not concatenation. We can use the existing `std::snprintf` calls already being used to format in the counts. Something like this
```suggestion
                    gettext("%lu standard %ssecurity updates"),
                    counts.standard,
                    lts_text
```

(This applies to all of the similar cases in this function)